guides/integrate-firebase.md

```diff
@@ -0,0 +1,346 @@
+---
+title: Firebase
+sidebar_position: 6
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import SetupJavaScript from '/docs/workflows/_sdk-setup/_setup-javascript.mdx';
+import SetupKotlin from '/docs/workflows/_sdk-setup/_setup-kotlin.mdx';
+import SetupSwift from '/docs/workflows/_sdk-setup/_setup-swift.mdx';
+import Arcade, {Clip} from '/src/components/Arcade.tsx';
+
+# Integrate Beyond Identity Passwordless Authentication with Firebase Authentication
+
+This guide provides information on how to set up Beyond Identity as a passwordless authentication provider for an application that uses [Firebase](https://firebase.google.com/docs/auth).
+
+This guide will cover:
+
+- How to configure Beyond Identity as an Identity Provider
+- How to create an identity and generate a passkey
+- How to authenticate with a passkey
+- How to configure the OpenID Connect plugin
+
+## Prerequisites
+
+- Set up a [developer account](/docs/v1/workflows/account-setup)
+
+## Firebase
+
+- [Authenticate Using OpenID Connect on Android](https://firebase.google.com/docs/auth/android/openid-connect)
+- [Authenticate Using OpenID Connect on Apple platforms](https://firebase.google.com/docs/auth/ios/openid-connect)
+
+## Install the Embedded SDK
+
+In order to use Beyond Identity functionality in your application, you will need to install the Beyond Identity SDK. The [Android SDK](https://github.com/gobeyondidentity/bi-sdk-android), the [Swift SDK](https://github.com/gobeyondidentity/bi-sdk-swift), and the [JavaScript SDK](https://github.com/gobeyondidentity/bi-sdk-js) provides functionality from passkey creation to passwordless authentication. A set of functions are provided to you through an `Embedded` namespace.
+
```

Comment:
The developer will need to install both a BI SDK and a Firefox SDK, right? I would list two sections for each under a general `Install SDKs` section.

```suggestion
## Install SDKs

### Install Firefox
....

### Install Beyond Identity SDK
In order to use Beyond Identity functionality in your application, you will need to install a Beyond Identity SDK. The [Android SDK](https://github.com/gobeyondidentity/bi-sdk-android), the [Swift SDK](https://github.com/gobeyondidentity/bi-sdk-swift), and the [JavaScript SDK](https://github.com/gobeyondidentity/bi-sdk-js) provides functionality from passkey creation to passwordless authentication. A set of functions are provided to you through an `Embedded` namespace.
```

Reply:
I made this change locally because the suggestion says 'Firefox' instead of 'Firebase' 🙂